symfit/core/fit.py

```diff
@@ -12,7 +11,8 @@
 from scipy.integrate import odeint
 
 from symfit.core.argument import Parameter, Variable
-from .support import seperate_symbols, keywordonly, sympy_to_py, cache, key2str
+from .support import \
+    seperate_symbols, keywordonly, sympy_to_py, cache, key2str, partial
```

Comment:
Why not import repeatable_partial (as partial)? This way we still end up with the partial from functools I think

Reply:
In `support.py`, I check for the python version. If this is py3.5 or later, `partial` is defined as `functools.partial`. Otherwise, this gives you `repeatable_partial` as `partial`.